format/FormatNexusEigerDLS16MI03.py

```diff
@@ -0,0 +1,46 @@
+import calendar
+
+import h5py
+
+from dxtbx.format.FormatCBFMiniPilatusHelpers import get_pilatus_timestamp
+from dxtbx.format.FormatNexusEigerDLS16M import FormatNexusEigerDLS16M
+from dxtbx.format.nexus import h5str
+
+
+def get_start_time(filename):
+    with h5py.File(filename, mode="r") as f:
+        timestamp = h5str(f["/entry/start_time"][()]).rstrip("Z")
+    return get_pilatus_timestamp(timestamp)
+
+
+class FormatNexusEigerDLS16MI03(FormatNexusEigerDLS16M):
```

Comment:
Please can you add a docstring explaining the context of this e.g. 

"Specific format overloading the module behaviour for the Eiger 2XE 16M on i03 for dates..."

For future reference